packages/hawtio/src/plugins/shared/operations/OperationForm.tsx

```diff
@@ -165,54 +167,47 @@ const OperationExecuteForm: React.FunctionComponent<{
   )
 }
 
-export const OperationForm: React.FunctionComponent<OperationFormProps> = props => {
-  const { selectedNode } = useContext(PluginNodeSelectionContext)
-  const { name, operation } = props
-  const [isExpanded, setIsExpanded] = useState(false)
+const OperationActions = ({
+  operation,
+  name,
+  objectName,
+}: {
+  operation: Operation
+  name: string
+  objectName: string
+}) => {
   const [isDropdownOpen, setIsDropdownOpen] = useState(false)
-
-  if (!selectedNode || !selectedNode.objectName || !selectedNode.mbean) {
-    return null
-  }
-
-  const { objectName } = selectedNode
-
-  const handleToggle = () => {
-    setIsExpanded(!isExpanded)
-  }
+  const [currentAlerts, setCurrentAlerts] = useState<React.ReactNode[]>([])
 
   const handleDropdownToggle = () => {
     setIsDropdownOpen(!isDropdownOpen)
   }
 
-  const copyMethodName = () => {
-    // TODO: impl
+  const copyToClipboard = (text: string) => {
+    navigator.clipboard.writeText(text)
+    setCurrentAlerts(prev => [
+      ...prev,
+      <Alert key={`{alert-${prev.length + 1}`} isLiveRegion title='Copied to clipboard' />,
+    ])
   }
 
-  const copyJolokiaURL = () => {
-    // TODO: impl
+  const copyMethodName = () => {
+    copyToClipboard(operation.readableName)
   }
 
-  const OperationCells = () => (
-    <DataListItemCells
-      dataListCells={[
-        <DataListCell key={`operation-cell-name-${name}`} isFilled={false}>
-          <code className='operation-datatype'>{operation.getReadableReturnType()}</code>
-          <b>{operation.readableName}</b>
-        </DataListCell>,
-        <DataListCell key={`operation-cell-desc-${name}`} isFilled={false}>
-          {operation.description}
-        </DataListCell>,
-      ]}
-    />
-  )
+  const copyJolokiaURL = async () => {
+    copyToClipboard(await operationService.getJolokiaUrl(objectName, name))
+  }
 
-  const OperationActions = () => (
+  return (
     <DataListAction
       id={`operation-actions-${name}`}
       aria-label={`operation actions ${name}`}
       aria-labelledby={`${name} operation-actions-${name}`}
     >
+      <AlertGroup isToast isLiveRegion>
```

Comment:
I just saw that we have an HawtIONotification element in the main page for this kind of toast alerts. Should I refactor this to use that better?